### PR TITLE
add info text when get long response api request

### PR DESCRIPTION
**Pull request by syauqy**
Hi @agallio, this is my attempt on the #44 issue.
I'm creating a simple timer using `useEffect` and `setTimeout` to check whether the loading passes 10secs and trigger the text visibility.
Let me know what do you think. Feel free to give any suggestions on a better approach. 🙏 

image

note: the gif above is not the updated version. But I've pulled the latest PR for this commit.

### CODE REVIEW

**Comment by agallio** [comment]
Hmmm I think it is better to calculate the duration from the data fetching method. See this [reference](https://stackoverflow.com/questions/61321105/measure-api-response-time-with-fetch-in-node-js).

**Comment by syauqy** [comment]
ahh, do you mean using console.time inside `useHospitalDataByProvince` while fetching the hospital data then passing it to `[province].js`? How about perf_hooks?

**Comment by agallio** [comment]
Yep. That's correct @syauqy 👍🏻 
Hmmm, I'm not familiar with `perf_hooks` yet. If the result is like what we intended, I think we can go with that.

**Comment by syauqy** [comment]
hi @agallio, I just know if swr has their own callback to handle this issue called `onSlowLoading` which trigger when the request time exceeded the `loadingTimeout` config/options. I'm using that callback to set if the state is `slowLoading` then render the text. Then using `onSuccess` callback to set the state back to `!slowLoading`. 
image


What do you think? I've pushed the change on the new commit.

File: pages/[province].js [hunk 1]
```diff
@@ -147,6 +150,12 @@ function ProvincePage(props) {
           ) : (
             <Box w="100%" textAlign="center">
               <Spinner size="lg" />
+              {slowLoading && (
+                <Text p={4} mt={4}>
+                  Pengambilan data Provinsi {getProvinceDisplayName(province)}{' '}
+                  membutuhkan waktu lebih lama. Mohon tunggu.{' '}
```
Comment (agallio) [inline pages/[province].js]: I think we can modify this to
"Pengambilan data untuk provinsi `<province>` membutuhkan waktu yang lebih lama. Mohon tunggu sesaat."

And please check the responsiveness too ya.

**Review by agallio — CHANGES_REQUESTED** [review]

File: pages/[province].js (hunk 1 above)
Comment (syauqy) [reply pages/[province].js]: Is this okay?
image

Comment (agallio) [reply pages/[province].js]: Oh sorry, could you add "" between the province name?

Comment (syauqy) [reply pages/[province].js]: Sure, pushed new commit  @agallio

**Review by agallio — APPROVED** [review]
LGTM 🚢 . Thanks @syauqy

**Pull request merged**